ansible/roles/sssd/tasks/configure.yml

```diff
@@ -30,5 +30,5 @@
 - name: "Ensure oddjob is started"
   service:
     name: oddjobd
-    state: "{{ sssd_enable_mkhomedir }}"
+    state: "{{ 'started' if sssd_enable_mkhomedir else 'stopped' }}"
```

Comment:
Was wondering if we should use when so that we never stop it:

```

        - name: "Ensure oddjob is started"
          service:
            name: oddjobd
            state: 'started'
            enabled: "{{ sssd_enable_mkhomedir }}"
      when: sssd_enable_mkhomedir | bool
  ```
Just because it may have some other uses but we definitely want it started if creating home directories